pkg/asset/ignition/bootstrap/content/bootkube_temporary.go

```diff
@@ -52,6 +53,27 @@ spec:
     targetPort: 53
 `))
 
+	BootkubeCheckpointerKubeConfig = template.Must(template.New("pod-checkpointer-kubeconfig").Parse(`
+apiVersion: v1
+kind: Secret
+metadata:
+  name: pod-checkpointer-kubeconfig
+  namespace: kube-system
+data:
+  kubeconfig: {{ .AdminKubeConfigBase64 }}
+`))
+
+	BootkubeKubeProxyKubeConfig = template.Must(template.New("kube-proxy-kubeconfig").Parse(`
```

Comment:
@squeed this is for you. What should we do here? The kube-proxy is temporary and is must be decoupled from the controller manager. Hence, this temporary file.